src/navigators/AppStack.js

```diff
@@ -17,17 +22,38 @@ const AppStack = () => (
     <Stack.Screen
       name={MAIN_SCREEN}
       component={MainScreen}
-      options={{ title: strings.MAIN_SCREEN.title }}
+      options={({ navigation }) => ({
+        title: strings.MAIN_SCREEN.title,
+        headerRight: () => (
+          <TouchableOpacity
+            onPress={() => navigation.push(CONVERSATIONS_SCREEN)}
+            style={styles.rightButton}>
+            <Image source={conversationsIcon} />
+          </TouchableOpacity>
+        ),
+      })}
     />
     <Stack.Screen
       name={CHAT_SCREEN}
       component={ChatScreen}
       options={({ route }) => ({
-        title: route.params.matchedUser.fullName,
+        title: route.params.userName,
         headerTintColor: BLACK,
         headerBackTitle: ' ',
-        // eslint-disable-next-line react/no-multi-comp
-        headerBackImage: () => <Image source={backIcon} style={{ marginLeft: 18 }} />,
+        headerBackImage: () => <Image source={backIcon} style={styles.backButtonMargin} />,
+      })}
+    />
+    <Stack.Screen
+      name={CONVERSATIONS_SCREEN}
+      component={ConversationsScreen}
+      options={({ navigation }) => ({
+        title: strings.CHAT.title,
+        headerRight: () => (
+          <TouchableOpacity onPress={() => navigation.pop()} style={styles.rightButton}>
+            <Image source={locationIcon} />
+          </TouchableOpacity>
```

Comment:
Same here, also you can make a `HeaderButton` common component and pass as a prop the icon you need to show, to reuse. WDYT?

Reply:
and the onPress navigation function, to go back or to go to some specific screen